src/main/java/org/apache/flink/benchmark/BlockingPartitionBenchmark.java

```diff
@@ -66,6 +66,16 @@ public void uncompressedMmapPartition(UncompressedMmapEnvironmentContext context
         executeBenchmark(context.env);
     }
 
+    @Benchmark
+    public void compressedSortShufflePartition(CompressedSortShuffleEnvironmentContext context) throws Exception {
+        executeBenchmark(context.env);
+    }
+
+    @Benchmark
+    public void uncompressedSortShufflePartition(UncompressedSortShuffleEnvironmentContext context) throws Exception {
+        executeBenchmark(context.env);
+    }
+
```

Comment:
nit: rename to `compressedSortPartition` and `uncompressedSortPartition` to shorten the benchmark name in the web UI?